doc/api/deprecations.md

```diff
@@ -95,12 +95,16 @@ methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
 <a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
+### DEP0007: Replace cluster worker.suicide with worker.exitedAfterDisconnect
 
-Type: Runtime
+Type: End-of-Life
 
-Within the `cluster` module, the [`worker.suicide`][] property has been
-deprecated. Please use [`worker.exitedAfterDisconnect`][] instead.
+In an earlier version of the Node.js `cluster`, a boolean property with the name
+`suicide` was added to the `Worker` object. The intent of this property was to
+provide an indication of how and why the `Worker` instance exited. In Node.js
+6.0.0, the old property was deprecated and replaced with a new
+[worker.exitedAfterDisconnect][] property. The old property name was inaccurate
```

Comment:
I think `inaccurate` is preferable to `misleading` here.

"The name is inaccurate" doesn't seem to be...uh...inaccurate. :-D

"The name is misleading" seems a bit loaded and I think less accurate.

Other options, although I'm fine with `inaccurate`:  `imprecise`, `confusing`, `difficult to construe`, `easy to misunderstand`